addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -467,7 +467,7 @@ def _process_accrual_plans(self, date_to=False, force_period=False, log=True):
         date_to = date_to or fields.Date.today()
         already_accrued = {allocation.id: allocation.already_accrued or (allocation.number_of_days != 0 and allocation.accrual_plan_id.accrued_gain_time == 'start') for allocation in self}
         first_allocation = _("""This allocation have already ran once, any modification won't be effective to the days allocated to the employee. If you need to change the configuration of the allocation, delete and create a new one.""")
-        for allocation in self:
+        for allocation in self.filtered(lambda a: a.allocation_type == 'accrual'):
```

Comment:
```suggestion
        for allocation in self:
            if allocation.allocation_type != 'accrual':
                continue
```

Reply:
Hello @ppr-odoo 

Thank you for your review; changes have been made.